apps/api/src/domain/models/term/entities/term.from-event-history.spec.ts

```diff
@@ -80,12 +91,37 @@ const audioAddedForTerm = termElicitedFromPrompt.andThen<AudioAddedForTerm>({
 
 const userId = buildDummyUuid(777);
 
+const buildTermFromEventHistory = (eventHistory: BaseEvent[], id: string) => {
+    const { creationEventHandlerMap, updateEventHandlerMap } = buildEventHandlerMaps([
+        // Creation Event Handlers
+        TermCreatedEventHandler,
+        PromptTermCreatedEventHandler,
+        // Update Event Handlers
+        TermTranslatedEventHandler,
+        ResourceReadAccessGrantedToUserEventHandler,
+        ResourcePublishedEventHandler,
+        TermElicitedFromPromptEventHandler,
+        AudioAddedForTermEventHandler,
+    ]);
+
+    const aggregateFactory = new CoscradEventSourcedAggregateFactory(
```

Comment:
It's not ideal that we have to do all of this for a simple "unit" test. It may be worth considering an alternative pattern that doesn't require this.